includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -322,7 +322,15 @@ private function courses_select( $courses, $selected_course, $select_id = 'cours
 			<option value="0"><?php echo esc_html( $select_label ); ?></option>
 			<?php
 			foreach ( $courses as $course ) {
-				echo '<option value="' . esc_attr( $course->ID ) . '"' . selected( $course->ID, $selected_course, false ) . '>' . esc_html( $course->post_title ) . '</option>';
+				$option_label = __( '(no title)', 'sensei-lms' );
+
+				if ( empty( $course->post_title ) ) {
+					$option_label = $option_label . ' ID: ' . $course->ID;
```

Comment:
Why not initializing the option_label here?

Maybe you could even do something like:
```
$option_label = empty( $course->post_title) ? __( '(no title)', 'sensei-lms') . ' ID: ' . $course-ID : $course->post_title;
```